schema.graphql

```diff
@@ -1,45 +1,36 @@
-type NewOwnership @entity(immutable: true) {
-  id: Bytes!
-  from: Bytes! # address
-  to: Bytes! # address
-  blockNumber: BigInt!
-  blockTimestamp: BigInt!
-  transactionHash: Bytes!
+type GlobalState @entity {
+  id: ID!
+  oracles: [Oracle!]! @derivedFrom(field: "state")
 }
 
-type NewPendingOwnership @entity(immutable: true) {
-  id: Bytes!
-  from: Bytes! # address
-  to: Bytes! # address
-  blockNumber: BigInt!
-  blockTimestamp: BigInt!
-  transactionHash: Bytes!
-}
-
-type OracleSet @entity(immutable: true) {
-  id: Bytes!
-  index: BigInt! # uint256
-  oracle: Bytes! # address
-  blockNumber: BigInt!
-  blockTimestamp: BigInt!
-  transactionHash: Bytes!
+type Oracle @entity {
```

Comment:
As discussed I would change this so that the id is not the index and we can preserve the history of oracles instead of overwriting it whenever there is a change. We can add an `active` boolean field to indicate wether or not the oracle is currently an active one.